### PR TITLE
Remove ELFs from access.rb and tmpl files.

### PR DESCRIPTION
**Pull request by tpendragon**
I need to get the fixture out of the full dump that I need, but my alma user doesn't have permission to run publishing profiles. I'm blocked until I get the MARC record for 99121576653506421.

Work towards #1203

### CODE REVIEW

**Review by christinach — APPROVED** [review]
This is ok but when we run to generate the tables form alma the elf will be again in the tmpl files.

**Comment by tpendragon** [comment]
> This is ok but when we run to generate the tables form alma the elf will be again in the tmpl files.

Yes, but the test will fail.

**Comment by christinach** [comment]
the elfs need to be removed from alma or  to exclude them in this method https://github.com/pulibrary/bibdata/blob/a973974e41920cbf4507d5f854e400519defd5b5/app/services/location_data_service.rb#L23 when we populate the tables. 
Your pr will work as long as we dont update the tables from alma which I will do when I update the flags for my pr. 
I'm ok if you want to go ahead and merge this and I will exclude the elfs in https://github.com/pulibrary/bibdata/pull/1213

File: spec/marc_to_solr/lib/config_spec.rb [hunk 1]
```diff
@@ -49,6 +49,15 @@ def fixture_alma_record(fixture_name)
       access_links = record["electronic_access_1display"]
       expect(JSON.parse(access_links.first)).to eq("http://dx.doi.org/10.1007/BFb0088073" => ["dx.doi.org"])
     end
+    it "does not index elf locations for alma" do
+      # I need a record where this would break without this change - probably 99121576653506421
```
Comment (eliotjordan) [inline spec/marc_to_solr/lib/config_spec.rb:53]: Don't need this comment anymore.

**Review by eliotjordan — CHANGES_REQUESTED** [review]
Small change requested, otherwise 👍

**Comment by christinach** [comment]
> > This is ok but when we run to generate the tables form alma the elf will be again in the tmpl files.
> 
> Yes, but the test will fail.

Yes you are right the test will fail.

**Comment by tpendragon** [comment]
To be clear, after deploying this we'll have to delete the DB entries for the elf locations from bibdata-alma.

**Comment by christinach** [comment]
I'm sorry @tpendragon this needs a rebase

**Comment by christinach** [comment]
It's ok to merge and have the test in place.

**Review by christinach — APPROVED** [review]

**Pull request merged**